client/src/components/my-mentor-card.tsx

```diff
@@ -327,9 +341,178 @@ export default function MyMentorCard(props: {
+            </Typography>
+            <Typography
+              variant="body1"
+              color="textSecondary"
+              align="left"
+              data-cy="mentor-card-scope-description"
+            >
+              {currentStage.description}
+            </Typography>
+            {props.type ? (
+              <Typography
+                variant="h6"
+                color="textSecondary"
+                align="left"
+                data-cy="mentor-card-type"
+              >
+                {props.type[0].toUpperCase() +
+                  props.type.slice(1).toLowerCase()}{" "}
+                Mentor
+              </Typography>
+            ) : (
+              <Typography
+                variant="h6"
+                color="textSecondary"
+                align="left"
+                data-cy="mentor-card-type"
+              >
+                Invalid Mentor
+              </Typography>
+            )}
+
+            <Typography
+              variant="body1"
+              color="textSecondary"
+              align="left"
+              data-cy="mentor-card-trained"
+            >
+              Last Trained: {props.lastTrainedAt.substring(0, 10)}
+            </Typography>
+          </Grid>
+
+          <Grid item={true} alignItems="center" xs={6}>
+            <Typography variant="body1" color="textSecondary">
+              Next Goal: {currentStage.next.name}
+              {"   "}
+              <Tooltip
+                title={
+                  <React.Fragment>
+                    <Typography color="inherit">
+                      {currentStage.next.name}
+                    </Typography>
+                    {currentStage.next.description}
+                  </React.Fragment>
+                }
+                data-cy="next-stage-info"
+              >
+                <HelpOutline fontSize="small" />
+              </Tooltip>
+            </Typography>
+
+            {currentStage.floor != 1000 && (
+              <StageProgress
+                value={props.value}
+                max={currentStage.max || 0}
+                percent={currentStage.percent || 0}
+              />
+            )}
+          </Grid>
+          <Grid
+            item={true}
+            alignItems="center"
+            xs={12}
+            md={6}
+            data-cy="modified-questions"
+          >
+            {difference < 0 ? (
+              <Typography>
+                You removed {-1 * difference} questions this session.
+              </Typography>
+            ) : (
+              <Typography>
+                You added {difference} questions this session.
+              </Typography>
+            )}
+          </Grid>
+        </Grid>
+      </CardContent>
+    </Card>
+  );
+}
+
 MyMentorCard.propTypes = {
   mentorId: PropTypes.string.isRequired,
   name: PropTypes.string.isRequired,
   value: PropTypes.number.isRequired,
   thumbnail: PropTypes.string.isRequired,
 };
+
+ProgressCard.propTypes = {
```

Comment:
I forget. Do propTypes do anything for us that a typescript interface does not? I guess the proptypes are checked at runtime and not just pre-build is the point?